api/src/Microsoft.Azure.IIoT.Api.Testing/cli/Program.cs

```diff
@@ -77,8 +75,6 @@ public class Program : IDisposable {
                 .AsImplementedInterfaces();
             builder.RegisterType<RegistryServiceClient>()
                 .AsImplementedInterfaces();
-            builder.RegisterType<VaultServiceClient>()
```

Comment:
`vault` should be removed from the comment on line `73`/`75`:

```
// Register twin, vault, and registry services clients
```

Reply:
Ah, the joy of copy paste :-)